client/modules/Slack/SlackBotSender.php

```diff
@@ -41,8 +41,8 @@ class SlackBotSender
      */
     public function __construct($slackBotUrl, $auth = null)
     {
-        $this->slackBotUrl = $slackBotUrl;
-        $this->auth        = $auth;
+        self::$slackBotUrl = $slackBotUrl;
```

Comment:
Yes, I also thought of that, but I thought you guys consciously made that decision, cause these properties are defined as static in the class. And the compiler was throwing notices. @l80